djangocms_googlemap/models.py

```diff
@@ -1,90 +1,288 @@
+# -*- coding: utf-8 -*-
+"""
+Enables the user to add various google maps plugin that adds maps or images
+from the google maps api.
+"""
+from __future__ import unicode_literals
+
+import re
+import json
+
 from django.db import models
-from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext, ugettext_lazy as _
 
 from cms.models import CMSPlugin
-from cms.utils.compat.dj import python_2_unicode_compatible
 
 
-@python_2_unicode_compatible
-class GoogleMap(CMSPlugin):
-    """
-    A google maps integration
-    """
-    translatable_content_excluded_fields = [
-        'address', 'zipcode', 'width', 'height']
+MAP_TYPES = ['ROADMAP', 'SATELLITE', 'HYBRID', 'TERRAIN']
```

Comment:
If these strings are needed for the API, can we also provide non-capitalised ones for the user to read?